src/Utilities/Compiler/Lightup/ITypeSymbolExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis;
+
+namespace Analyzer.Utilities.Lightup
+{
+    internal static class ITypeSymbolExtensions
```

Comment:
I presume these are the same light-up based extensions that would be needed in CodeStyle layer in Roslyn?

Reply:
Can we create a tracking work item to delete all light-up based code once we move to MS.CA package that has these public APIs?